17 error Error: Command failed with exit code 1: git remote update
17 error fatal: unable to access 'https://github.com/yhyancy/TestLerna.git/': Failed to connect to github.com port 443 after 75025 ms: Operation timed out
17 error     at makeError (/Users/yancy/Documents/YINKE_Learning/Lesson02/node_modules/.pnpm/execa@5.0.0/node_modules/execa/lib/error.js:59:11)
17 error     at module.exports.sync (/Users/yancy/Documents/YINKE_Learning/Lesson02/node_modules/.pnpm/execa@5.0.0/node_modules/execa/index.js:186:17)
17 error     at Object.execSync (/Users/yancy/Documents/YINKE_Learning/Lesson02/node_modules/.pnpm/@lerna+child-process@7.1.1/node_modules/@lerna/child-process/index.js:39:16)
17 error     at updateRemote (/Users/yancy/Documents/YINKE_Learning/Lesson02/node_modules/.pnpm/lerna@7.1.1/node_modules/lerna/dist/index.js:7619:18)
17 error     at isBehindUpstream (/Users/yancy/Documents/YINKE_Learning/Lesson02/node_modules/.pnpm/lerna@7.1.1/node_modules/lerna/dist/index.js:7609:3)
17 error     at VersionCommand.initialize (/Users/yancy/Documents/YINKE_Learning/Lesson02/node_modules/.pnpm/lerna@7.1.1/node_modules/lerna/dist/index.js:7919:57)
